remote-api/src/test/java/org/alfresco/rest/api/impl/rules/RuleSetsImplTest.java

```diff
@@ -130,4 +139,14 @@ public void testGetRuleSetById()
 
         assertEquals(RuleSet.of(RULE_SET_ID), actual);
     }
+
+    @Test
+    public void testLoadRuleSet()

```

Comment:
I think (as stated in previous comment) this test should be split into at least 2 hitting against `getRuleSetById` and `getRuleSets` methods with regards to a different `includes` list parameters (assuming that this list will also grow in the future).

Reply:
I agree that the method will grow, and so we'll want more tests in the future.  I don't think it makes sense to test it twice for each new includes field though.  I'll move the method into a new class and we can unit test it separately.